unicodetools/src/main/java/org/unicode/text/UCD/TestUnicodeInvariants.java

```diff
@@ -691,26 +690,69 @@ private static void inLine(
     }
 
     // A one-token lookahead.
-    // Tokens are defined as runs of [^\p{Pattern_White_Space}\p{Pattern_Syntax}],
-    // or single code points in \p{Pattern_Syntax}.
+    // Tokens are defined as:
+    // 1. words: runs of [^\p{Pattern_White_Space}\p{Pattern_Syntax}];
+    // 2. simple operators: sequences of the form \p{Pattern_Syntax} \p{Mn}*;
+    // 3. explicitly expected sequences of words and simple operators without intervening spaces;
+    //    this allows for contextually accepting operators such as :=, >>, ’s, or .GT.,
+    //    without treating, e.g., every >> as atomic.
     private static class Lookahead {
```

Comment:
Nit: It would be nice to use `/** ... */` doc strings so that these docs show up in an IDE.